src/components/TimeLine/time-line.js

```diff
@@ -53,6 +59,15 @@ const graphColors = {
   }
 };
 
+const mapDispatchToProps = dispatch => {
+  return {
+    showYearDetails: year => {
```

Comment:
make sure there is alway only one selected Something in the main state! 
otherwise the state cannot be reproduced from the URL since it is unclear what the sidebar needs to show.
So before setting a selected anything, use the 'deselect Items' action to clear all selected items.
this should include 'selected Year'

Reply:
actually, while this is true, it's a bad solution overall right now.
I'll make a note to rewrite this to be automatically done any time some new highlighted component is set (no need to dispatch a second action if it is done in any case)

Reply:
ok, fixed it with the deselectItems temporarily